samples/excel/85-preview-apis/comment-basics.yaml

```diff
@@ -1,28 +1,25 @@
-order: 3
-id: excel-comment
-name: Comments
-description: 'Adds, edits, and removes comments and comment replies.'
+order: 2
+id: excel-comment-basics
+name: Add and remove comments
```

Comment:
Since all the other comment-related snippet names start with the word "Comment" -- perhaps consider naming this one "Comment basics" (for consistency in how the snippets look when grouped together in the samples menu)?

*Note* - if you do rename this one, you should also change `order` value to `3` (and set the `order` value to `2` in `samples/excel/85-preview-apis/event-column-and-row-sort.yaml`)

Reply:
These will be moved under a "Comments" section in the samples once they're out of preview.  Does that change you suggestion at all?

Reply:
Good point. FWIW, I still prefer "Comment basics" for its simplicity and parallelism in wording with other comment-related snippets...and because the description that appears directly below the snippet title in the samples menu will be redundant with the current name. That said, please do as you see fit here :)